examples/gogo/command/src/main/java/org/osgi/enroute/examples/gogo/command/ComponentSingle.java

```diff
@@ -0,0 +1,20 @@
+
+package org.osgi.enroute.examples.gogo.command;
+
+import org.osgi.service.component.annotations.Component;
+
+@Component(
+        /* Activate the component immediately, also if no one depends on it. */
+        immediate = true,
+        /* You need to implement some class, without any it will not be picked up. */
```

Comment:
As per the Multi-Command, this should not be immediate, and the comment information belongs in a web page describing the example.

Reply:
If the examples are not about all possible technical options but only about showing users a good practice, there does not need to exist separate examples for single and multiple functions at all.
The "normal" consumer of that examples (if my reading has been correct) don't care about string array property and string property, but only about a working example to copy and paste.

So, the single can be removed and the multi could be renamed.